analyzers/src/SonarAnalyzer.CSharp/Rules/MultilineBlocksWithoutBrace.cs

```diff
@@ -96,32 +97,32 @@ private static StatementSyntax LastStatementInIfChain(IfStatementSyntax ifStatem
 
     private static void CheckStatement(SyntaxNodeAnalysisContext context, StatementSyntax first, string executed, string execute)
     {
-        if (SecondStatement(context.Node, first) is { } second && IsNotEmpty(first) && IsNotEmpty(second))
+        if (IsNotEmpty(first)
+            && SecondStatement(context.Node, first) is { } second
+            && IsNotEmpty(second)
+            && InsufficientIndenting(first, second))
         {
-            var firstPosition = ProxyStartPosition(first);
-            var secondPosition = StartPosition(second);
-
-            if (secondPosition.Character > firstPosition.Character - IndentSize)
-            {
-                var lineSpan = context.Node.SyntaxTree.GetText().Lines[secondPosition.Line].Span;
-                var location = Location.Create(context.Node.SyntaxTree, TextSpan.FromBounds(second.SpanStart, lineSpan.End));
-                var blockSize = secondPosition.Line - StartPosition(first).Line + 1;
-                var additional = new[] { first.GetLocation() };
-                context.ReportIssue(Rule.CreateDiagnostic(context.Compilation, location, additional, executed, execute, blockSize));
-            }
+            var secondLine = StartPosition(second).Line;
+            var lineSpan = context.Node.SyntaxTree.GetText().Lines[secondLine].Span;
+            var location = Location.Create(context.Node.SyntaxTree, TextSpan.FromBounds(second.SpanStart, lineSpan.End));
+            var blockSize = secondLine - StartPosition(first).Line + 1;
+            var additional = new[] { first.GetLocation() };
+            context.ReportIssue(Rule.CreateDiagnostic(context.Compilation, location, additional, executed, execute, blockSize));
         }
     }
 
     private static bool IsNotEmpty(SyntaxNode node) =>
         node is not EmptyStatementSyntax;
 
-    private static LinePosition ProxyStartPosition(SyntaxNode node)
+    private static bool InsufficientIndenting(SyntaxNode first, SyntaxNode second)
```

Comment:
Is `insufficient` the correct word here? Insufficient sounds to me that you need to add more indenting to fix it. While that's what we're trying to prevent.
What about `WrongIndenting`?

Reply:
I renamed it to `MisleadingIndenting()`, I think that name is even better.